### PR TITLE
UQ-188 Upgraded to Ucommerce v9.4

### PR DESCRIPTION
**Pull request by timgaunt**

### CODE REVIEW

**Comment by timgaunt** [comment]
Please release this nuget package once reviewed so we can update Sitefinity Quantum

File: Ucommerce.Sitefinity.UI/packages.config
Comment (skovsboll) [inline Ucommerce.Sitefinity.UI/packages.config:49]: This is a huge commit. And most files only changed because `Ucommerce.Sitefinity`, a dependency in packages.config, installed them. 

I'd love to not have all of those files and dirs inside the `Ucommerce.Sitefinity.UI/Ucommerce` dir under source control (twice). They're really part of the main Ucommerce repository. 

If we could remove the `Ucommerce.Sitefinity` package and keep the `Ucommerce` and `Ucommerce.Core` packages, it would make future maintenance so much easier.

File: README.md
Comment (skovsboll) [inline README.md:30]: Great addition! :+1:

**Review by skovsboll — CHANGES_REQUESTED** [review]
Hi Tim, thanks for doing this, and sorry for the delay.
Before we merge this, I'd like to see if we can slim down the number of files committed. Most of the 49 files changed are not needed in this repo at all. See my comment for `packages.config` at the bottom.

File: Ucommerce.Sitefinity.UI/Mvc/Model/FacetsFilterModel.cs
Comment (skovsboll) [inline Ucommerce.Sitefinity.UI/Mvc/Model/FacetsFilterModel.cs:100]: I'd love to see a unit test for this.
The commit is called `Ignored all PriceGroups that aren't the current currency` but I don't see any reference to the current currency in the code. Anyone reading the code in the future will also wonder what that line does.

File: Ucommerce.Sitefinity.UI/Mvc/Model/FacetsFilterModel.cs
Comment (skovsboll) [inline Ucommerce.Sitefinity.UI/Mvc/Model/FacetsFilterModel.cs:98]: Why load the pricegroup from the db here? We already have `pricegroup.Name` and that's the only property we ever use.

**Review by skovsboll — COMMENTED** [review]
I have a few concerns about the `UQ-188 Ignored all PriceGroups that aren't the current currency` commit.

**Pull request merged**
